docs/niscope/session.rst

```diff
@@ -410,6 +410,8 @@ niscope.Session
    +-------------------------------------------------------+
    | :py:func:`fetch`                                      |
    +-------------------------------------------------------+
+   | :py:func:`fetch`                                      |
```

Comment:
Seems like the name rendered wrong, should be fetch_array. Something to look into.
Should I open an issue?

Reply:
This doesn't do anything with the documentation. It will be wrong for now.